docs/sources/datasources/tempo/query-editor/_index.md

```diff
@@ -123,15 +123,15 @@ Using the service graph view, you can:
 - Determine how long the slowest queries in your service take to complete.
 - Examine all traces that contain spans of particular interest based on rate, error, and duration values (RED signals).
 
-For more information about the service graph, refer to [Service graph](../service-graph/).
+For more information about the service graph, refer to [Service graph](../service-graph.md).
```

Comment:
For more information about the service graph, refer to [Service graph](../service-graph/)

Reply:
Hi, @irenerl24,

Oh, I did not know about the links.
Thank you .

Cheers